app/models/video.rb

```diff
@@ -16,7 +16,7 @@ def self.search_by_title(query = '')
 
   # returns the rating-average from all reviews
   def avg_rating
-    ratings = reviews.map(&:rating)  
+    ratings = reviews.map(&:rating).reject(&:nil?)
```

Comment:
You could also use `compact` instead of iterating the array again using `reject`.